core/arch/arm/kernel/thread_private.h

```diff
@@ -165,13 +165,15 @@ extern thread_pm_handler_t thread_system_reset_handler_ptr;
  */
 extern long thread_user_kcode_offset;
 
-void thread_vect_table(void);
-
 /*
  * Initializes VBAR for current CPU (called by thread_init_per_cpu()
  */
 void thread_init_vbar(void);
 
+void thread_excp_vect(void);
+void thread_excp_vect_workaround(void);
+void thread_excp_vect_workaround_a15(void);
```

Comment:
Actually, patch "core: rename exception vectors"  does two thing: renames vectors and adds global definitions for them.

I think you need to cover this in the commit message, or to do this in separate changes.

Reply:
OK, I'll update the commit message